src/server/v2.0/handler/permissions.go

```diff
@@ -16,11 +16,12 @@ package handler
 
 import (
 	"context"
-
+	
```

Comment:
heading white space should be removed
 (gogofmt -w src/server/v2.0/handler/permissions.go )

Reply:
resolved.